docs: update News Tutorial

kenjis opened the pull request:
**Description**
- use `$this->validate()` and `Validation::getValidated()` #7420
- add `News::new()` to show HTML form
- change `News::view()` to `show()`
- change routing
- add section titles

Before:
```
+--------+--------------+------+---------------------------------+----------------+---------------+
| Method | Route        | Name | Handler                         | Before Filters | After Filters |
+--------+--------------+------+---------------------------------+----------------+---------------+
| GET    | /            | »    | \App\Controllers\Home::index    |                | toolbar       |
| GET    | news/create  | »    | \App\Controllers\News::create   |                | toolbar       |
| GET    | news/([^/]+) | »    | \App\Controllers\News::view/$1  |                | toolbar       |
| GET    | news         | »    | \App\Controllers\News::index    |                | toolbar       |
| GET    | pages        | »    | \App\Controllers\Pages::index   |                | toolbar       |
| GET    | ([^/]+)      | »    | \App\Controllers\Pages::view/$1 |                | toolbar       |
| POST   | news/create  | »    | \App\Controllers\News::create   | csrf           | toolbar       |
+--------+--------------+------+---------------------------------+----------------+---------------+
```

After:
```
+--------+--------------+------+---------------------------------+----------------+---------------+
| Method | Route        | Name | Handler                         | Before Filters | After Filters |
+--------+--------------+------+---------------------------------+----------------+---------------+
| GET    | /            | »    | \App\Controllers\Home::index    |                | toolbar       |
| GET    | news         | »    | \App\Controllers\News::index    |                | toolbar       |
| GET    | news/new     | »    | \App\Controllers\News::new      |                | toolbar       |
| GET    | news/([^/]+) | »    | \App\Controllers\News::show/$1  |                | toolbar       |
| GET    | pages        | »    | \App\Controllers\Pages::index   |                | toolbar       |
| GET    | ([^/]+)      | »    | \App\Controllers\Pages::view/$1 |                | toolbar       |
| POST   | news         | »    | \App\Controllers\News::create   | csrf           | toolbar       |
+--------+--------------+------+---------------------------------+----------------+---------------+
```
**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

michalsn reviewed (approved):
I find it valuable that we were using `$this->request->is('post')` in the previous version, but it's not a deal breaker for me. Nice update.

kenjis commented:
The `create()` method and its explanation were too long and it was difficult to read, so I split it into two methods and defined GET and POST routes. 

Then `$this->request->is()` is no longer needed. It would be strange to keep it, so I removed it.
